tools/picard.wdl

```diff
@@ -306,3 +306,85 @@ task clean_sam {
         bam: "Input BAM format file to clean"
     }
 }
+
+#ID:2860235881 
+#PL:illumina
+#PU:622BR.8
+#LB:H_LC-SJRB001-X-SJ39.3-8L-lib4
+#PI:309
+#DS:paired end
+#DT:2010-07-12T19:00:00-0500
+#SM:H_LC-SJRB001-X-SJ39.3-8L
+#CN:WUGSC
+#PG:bwa
+task add_or_replace_read_groups {
```

Comment:
The underlying Picard function is named "AddOrReplaceReadGroups". It is somewhat counter-intuitive because it assigns all reads in the file to the newly specified read group. It either adds it to the BAM or replaces the existing read group specifications.

Reply:
Ok that makes sense. I'd make the meta section explicit about what happens in this task.